public/components/application-ui-toggles/5.html

```diff
@@ -0,0 +1,22 @@
+<label for="AcceptConditions" class="absolute cursor-pointer">
+
+  <input type="checkbox" id="AcceptConditions"
+    class="sr-only [&:checked_+_span_svg[data-checked-icon]]:block [&:checked_+_span_svg[data-unchecked-icon]]:hidden" />
```

Comment:
I think there's a cleaner way of doing this, something like `peer-checked` springs to mind.